x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/create_datasource_page/components/endpoint_configuration.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { useLocation } from 'react-router-dom';
+import { EuiLink } from '@elastic/eui';
+
+export const EndpointConfiguration = memo<{ editMode: boolean }>(({ editMode }) => {
+  const pathname = useLocation().pathname.split('/');
+  const policyId = pathname[pathname.length - 1];
+  const linkToSiemApp = `/app/siem#/policy/${policyId}`;
+  return (
+    <>
+      {editMode === true ? (
+        <>
+          <FormattedMessage
+            id="xpack.ingestManager.editDatasource.stepConfigure.endpointConfiguration"
+            defaultMessage="See security app policy tab for additional configuration options: "
+          />
+          <EuiLink href={linkToSiemApp}>Click me to configure</EuiLink>
```

Comment:
Since we'll be switching apps here, I think we have do a little bit more to avoid the full app refresh.  

@paul-tavares does something similar here https://github.com/elastic/kibana/blob/master/x-pack/plugins/siem/public/endpoint_policy/view/policy_list.tsx#L137